include/sir/config.h

```diff
@@ -378,11 +378,9 @@
 # endif
 
 /** The maximum size, in characters, of final formatted output. */
-# if !defined(SIR_MAXOUTPUT)
-#  define SIR_MAXOUTPUT \
-     (SIR_MAXMESSAGE + (SIR_MAXSTYLE * 2) + SIR_MAXTIME + SIR_MAXLEVEL + \
-         SIR_MAXNAME + (SIR_MAXPID   * 2) + SIR_MAXMISC + 1)
-# endif
+# define SIR_MAXOUTPUT \
+    (SIR_MAXMESSAGE + (SIR_MAXSTYLE * 2) + SIR_MAXTIME + SIR_MAXLEVEL + \
+        SIR_MAXNAME + (SIR_MAXPID   * 2) + SIR_MAXMISC + 2)
```

Comment:
Sorry, I didn't explain this well. it should end with `+ 2 + 1` (CRLF + \0)

Reply:
oic, aight